### PR TITLE
Cosmetic gas mask

### PR DESCRIPTION
**Pull request by EthanRockss**
## About The Pull Request
Adds two prop masks of the default gas mask and the atmos gas mask. Both masks are purchasable from the autodrobe. 

## Why It's Good For The Game
Restores the gas masks previous cosmetic glory without the cone. The drawback being not filtering gas or protect against pepper spray. With the addition of being able to purchase a prop atmos mask from the autodrobe, it will result in players not breaking into atmos to steal their masks. I have also been told that Oranges said that he would be okay with something like this a while ago in a q/a.
example

## Changelog

:cl:
add: Added prop gas masks for both default and atmos
add: Added gas masks to autodrobe
/:cl:

### CODE REVIEW

**Comment by moocowswag** [comment]
I`m not against this but won`t people be unable to tell if the person wearing the gas mask has protection or not.

**Comment by Farquaar** [comment]
> Im not against this but wont people be unable to tell if the person wearing the gas mask has protection or not.

When you examine them it'll say they're wearing a prop gas mask. It's kind of like the toy eswords and laser tag guns- it looks effective until you shift-click the guy holding it.

File: code/modules/clothing/masks/gasmask.dm [hunk 1]
```diff
@@ -386,3 +386,25 @@
 	flags_inv = HIDEFACIALHAIR
 	w_class = WEIGHT_CLASS_NORMAL
 	inhand_icon_state = "driscoll_mask"
+
+/obj/item/clothing/mask/gas/prop
+	name = "prop gas mask"
+	desc = "A prop gas mask designed for appearance. Unlike a normal gas mask this does not filter gasses or protect against pepper spray."
+	icon_state = "gas_alt"
+	flags_inv = HIDEEARS|HIDEEYES|HIDEFACE|HIDEFACIALHAIR|HIDESNOUT
```
Comment (Ghilker) [inline code/modules/clothing/masks/gasmask.dm:394]: Shouldn't have those if it's just a prop

File: code/modules/clothing/masks/gasmask.dm [hunk 2]
```diff
@@ -386,3 +386,25 @@
...
+	w_class = WEIGHT_CLASS_SMALL
+	inhand_icon_state = "gas_alt"
+	permeability_coefficient = 0.5
+	flags_cover = MASKCOVERSEYES
```
Comment (Ghilker) [inline code/modules/clothing/masks/gasmask.dm:398]: And this

**Comment by realthomas69** [comment]
finally

**Comment by optimumtact** [comment]
it need some kind of distinguishing feature like a colour stripe

**Comment by SinfulBliss** [comment]
> it need some kind of distinguishing feature like a colour stripe

LET US HAVE OUR SOUL BACK ORANGEMAN

**Comment by realthomas69** [comment]
> it need some kind of distinguishing feature like a colour stripe

i dunno, wouldnt that look a little odd? i dont think its a big deal anyway, theres already lots of items like this

File: code/modules/clothing/masks/gasmask.dm (hunk 2 above)
Comment (EthanRockss) [reply code/modules/clothing/masks/gasmask.dm:398]: Removing the flags_cover will change it back to being pepper proof

File: code/modules/clothing/masks/gasmask.dm (hunk 1 above)
Comment (EthanRockss) [reply code/modules/clothing/masks/gasmask.dm:394]: I'm a little confused are you saying I should remove those including the name and description?

**Comment by SinfulBliss** [comment]
> > it need some kind of distinguishing feature like a colour stripe
> 
> wouldnt that look a little odd? i dont think its a big deal anyway, theres already lots of items like this, like farquaar said

That's true there's stuff like tacticool turtleneck, tactical snack rig, syndie replica helmet/suit and toy energy swords which look identical but upon examine will show their true nature. An examine should be enough IMO.

**Comment by optimumtact** [comment]
this wasn't a debate

File: code/modules/clothing/masks/gasmask.dm (hunk 1 above)
Comment (Ghilker) [reply code/modules/clothing/masks/gasmask.dm:394]: No wait not these, these are for sprite, resolve this one

File: code/modules/clothing/masks/gasmask.dm (hunk 2 above)
Comment (Ghilker) [reply code/modules/clothing/masks/gasmask.dm:398]: Set it to NONE

**Comment by necromanceranne** [comment]
Just add a little orange stripe under the eyes or something

**Comment by Serxule** [comment]
dont add any stripe, this is the exact same as toy eswords and the fake syndi spacesuit

**Comment by wesoda25** [comment]
Most people will likely not bother to verify whether your mask is real or fake and just treat it like it is, meaning they won't try pepperspray. So people with these would reap some of the benefits of masks without actually needing them. Not having FOV means these could actually be advantageous in certain situations. Plus it conceals your whole face.

**Comment by SinfulBliss** [comment]
> Most people will likely not bother to verify whether your mask is real or fake and just treat it like it is, meaning they won't try pepperspray. So people with these would reap some of the benefits of masks without actually needing them. Not having FOV means these could actually be advantageous in certain situations. Plus it conceals your whole face.
The concealing part was a mistake, Ghilker already pointed that out in the fix. Also I speak for everyone when I say - people would rather gasmasks not give any pepperspray benefit, _or_ gas filter, or any benefit whatsoever, simply to be able to wear them again without FoV. Peeps want their gasmasks back, even if they're totally useless and have no upside at all, we just love the style and it's become a staple of assistants for many years. 

I think Skelethan is working to get someone to make the differentiable from their real counterparts, and the conceal would be fixed too.

**Comment by Mothblocks** [comment]
Toy eswords have visibly lower brightness and are used by people to playfully attack others, with a very noticable faint tap sound, without any examine or extra work.

Laser tag guns tell you they are laser gun beams in chat without you having to look for it. A bit closer to nichety, but still not requiring extra work. This is also something used by people to playfully attack others.

This is a passive item that people will exclusively use to have equipped, separating it from the two. There is zero noticable tell unless you examine the person (active action), or unless they are wearing a mismatch ID, in which you can tell they are disguised. The comparison is very very weak.

**Comment by Mothblocks** [comment]
Frankly I think the funniest way to make this work is make this have the eyes cut out, then people can find like green glasses or something to make something that looks like a dopey, but still obviously fake, gas mask.

File: code/modules/clothing/masks/gasmask.dm [hunk 3]
```diff
@@ -386,3 +386,17 @@
 	flags_inv = HIDEFACIALHAIR
 	w_class = WEIGHT_CLASS_NORMAL
 	inhand_icon_state = "driscoll_mask"
+
+/obj/item/clothing/mask/gas/prop
+	name = "prop gas mask"
+	desc = "A prop gas mask designed for appearance. Unlike a normal gas mask this does not filter gasses or protect against pepper spray."
+	flags_cover = NONE
```
Comment (Fikou) [inline code/modules/clothing/masks/gasmask.dm:393]: i think you want maskcoversmouth here, weird that people would be able to eat with it

**Comment by tralezab** [comment]
> Frankly I think the funniest way to make this work is make this have the eyes cut out, then people can find like green glasses or something to make something that looks like a dopey, but still obviously fake, gas mask.

agree 100% but people want an exact sprite

**Comment by MetalClone** [comment]
It's not the biggest deal to have look-alike content, Toy Syndicate soft space suits are a good example of this. Plus, it's always worth flashing/pepperspraying people just in case they don't actually have real protections.

**Comment by EthanRockss** [comment]
example1
example2
Would making the transparency of the lenses be enough change? Mask on left is with lower transparency right is normal mask.

**Comment by optimumtact** [comment]
can you take a picture in game with the full client screen? it's hard to judge with those images how visible it is during normal gameplay, which is what matters.

**Comment by EthanRockss** [comment]
> can you take a picture in game with the full client screen? it's hard to judge with those images how visible it is during normal gameplay, which is what matters.

picture

**Comment by optimumtact** [comment]
ok I wasn't clear enough sorry, that's on me, I mean a side by side with the old and new if possible.

If not I get it, I'll checkout the fork and look myself

**Comment by EthanRockss** [comment]
image

**Comment by optimumtact** [comment]
thanks, they look okayish to me, @Mothblocks thoughts? might still be too subtle

**Comment by Mothblocks** [comment]
"Okayish" is the best way to describe it. The atmos mask is obvious, not so sure about the standard gas mask.

**Comment by Mothblocks** [comment]
Random thought: What if the sides are kept like that (distinct, but not super obvious), but the sides of the mask are more pushed forward?

**Comment by EthanRockss** [comment]
image
Here's some changes. Left is default, middle is more transparent, and right is a bluer lens.

**Comment by Mothblocks** [comment]
Far right is good :+1:

**Comment by EthanRockss** [comment]
> Far right is good 👍
Alright so blue lenses for regular mask, the modified atmos mask is good as well?

**Comment by Mothblocks** [comment]
Yeah should be fien

**Pull request merged**